example/app/data/DemoDataSource/recipes/plugins/generic-list/orderItem.json

```diff
@@ -21,103 +21,6 @@
             },
             "scope": "self"
           }
-        },
```

Comment:
The old config was wrong, since the orderItem is NOT a list it does not make sense to define list plugins in the initial UI recipe